pyclaragenomics/test/test_cudapoa_bindings.py

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+#
+# NVIDIA CORPORATION and its licensors retain all intellectual property
+# and proprietary rights in and to this software, related documentation
+# and any modifications thereto.  Any use, reproduction, disclosure or
+# distribution of this software and related documentation without an express
+# license agreement from NVIDIA CORPORATION is strictly prohibited.
+#
+
+from difflib import SequenceMatcher
+import pytest
+import random
+
+from claragenomics.bindings.cudapoa import CudaPoaBatch
+import claragenomics.bindings.cuda as cuda
+
+
+@pytest.mark.gpu
+def test_cudapoa_simple_batch():
+    (free, total) = cuda.cuda_get_mem_info(cuda.cuda_get_device())
+    batch = CudaPoaBatch(10, 0.9 * free)
```

Comment:
the `0.9 * free` isn't very user-friendly, can we make batch initialise to this by default?

Reply:
The main reason I didn't want to make any default for memory if someone uses multiple batches to play around, then it'll run into out of memory issues